wiki/Main_Page/es.md

```diff
@@ -1,22 +1,19 @@
 ---
-no_native_review: true
 layout: main_page
-outdated_translation: true
-outdated_since: 1da34ad18aa8565e779e579594fa8c606ee00808
 ---
 
 <!-- Do not add any empty lines inside this div. -->
 
 <div class="wiki-main-page__blurb">
-Bienvenido a la wiki de osu!, la base de conocimientos de código abierto que contiene una amplia variedad de información relacionada con osu!, un juego de ritmo gratuito.
+Bienvenido a la osu! wiki, la base de conocimientos de código abierto que contiene una amplia gama de información relacionada con osu!, un juego de ritmo gratuito.
 </div>
 
 <div class="wiki-main-page__panels">
 <div class="wiki-main-page-panel wiki-main-page-panel--full">
 
 # Para empezar
 
-Artículos y guías importantes para ayudarte a sumergirte en osu!. En caso de problema, no dude en consultar el [foro de ayuda](https://osu.ppy.sh/forum/5).
+Artículos y guías importantes que te ayudarán a sumergirte en osu!. En caso de que algo salga mal o parezca incierto, asegúrese de visitar el [foro de ayuda](https://osu.ppy.sh/forum/5).
```

Comment:
I think the usage of `asegúrese` is a bit odd. This is because `que te ayudarán` references the user directly while `asegúrese` references anyone.

```suggestion
Artículos y guías importantes que te ayudarán a sumergirte en osu!. En caso de que algo salga mal o parezca incierto, visita el [foro de ayuda](https://osu.ppy.sh/forum/5).
```